samples/qa-i18n-async/package-lock.json

```diff
@@ -1,13 +1,420 @@
 {
 	"name": "qa-i18n-async",
-	"version": "1.0.0",
+	"version": "0.1.0",
```

Comment:
🤔  Why does this sample have package-lock? Should we remove it like others?

Reply:
I agree, I will add a new commit that removing this file